rollup.config.js

```diff
@@ -1,18 +1,33 @@
 import typescript from "rollup-plugin-typescript2";
 import { terser } from "rollup-plugin-terser";
+import replace from "@rollup/plugin-replace";
 
 function config({ format, minify, input, ext = "js" }) {
-  const dir = `dist/${format}/`;
+  const dir = `dist/${format}`;
   const minifierSuffix = minify ? ".min" : "";
+  const workerThreadsImport = 'await import("worker_threads")';
   return {
     input: `./src/${input}.ts`,
     output: {
       name: "Comlink",
       file: `${dir}/${input}${minifierSuffix}.${ext}`,
       format,
       sourcemap: true,
+      inlineDynamicImports: true,
     },
+    external: ["worker_threads"],
     plugins: [
+      replace({
```

Comment:
This does the replacement of the TLA dynamic import

Reply:
Is this still needed today with TLA support in Node since 14.8, seems like we could drop this?